ContextDesktop/src/test/java/com/github/migi_1/Context/model/TestEnvironment.java

```diff
@@ -29,44 +30,49 @@
  * Test suite for the Environment class.
  */
 @RunWith(PowerMockRunner.class)
-@PrepareForTest({ProjectAssetManager.class})
+@PrepareForTest(fullyQualifiedNames = "com.github.migi_1.Context.*")
 public class TestEnvironment {
 
 	protected Environment environment;
-	
+
 	protected Node root;
 	protected AssetManager assetManager;
-	
+	private HUDController hudController;
+
+
 	/**
 	 * Initialises the environment field for testing.
+	 * @throws Exception exception that is thrown.
 	 */
 	@Before
-	public void setUp() {
+	public void setUp() throws Exception {
 		this.environment = new Environment();
-		
+
+        hudController = Mockito.mock(HUDController.class);
```

Comment:
The indenting doesn't match here. There's spaces in front of line 51 and 55, while the other lines have tabs.